packages/mobile/components/ToastContainer.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="typescript">
+    import { Alert, Toast } from 'shared/components'
+    import { notifications } from '@auxiliary/notification'
+    import { fade } from 'svelte/transition'
+
+    $: toasts = $notifications.map((notification) => ({
+        type: notification.type,
+        alert: notification.alert,
+        message: notification.message,
+        subMessage: notification.subMessage,
+        progress: notification.progress,
+        actions: notification.actions.map((action, actionIndex) => ({
+            ...action,
+            onClick: () => action.callback(notification, actionIndex),
+        })),
+    }))
+</script>
+
+<toast-container class="fixed top-0 flex flex-col z-10 w-full p-5">
```

Comment:
This div has a parasite padding that doesnt allow clicking on the back arrows 
![image](https://user-images.githubusercontent.com/3624944/214266862-a58f3933-62f6-4d2e-9996-36931e432a42.png)

Reply:
it also affects the top bar of the wallet, i would update it so when there are no notifications, the containers takes zero space